spec/requests/create_heritage_spec.rb

```diff
@@ -105,5 +110,16 @@
       let(:version) { 2 }
       it_behaves_like "create"
     end
+
+    context "when the endpoint is not belonged to heritage" do
```

Comment:
Is this what you mean?

```suggestion
    context "when the endpoint does not belong to a district " do
```

Reply:
ah thanks!